partiql-tests-data/eval/misc.ion

```diff
@@ -0,0 +1,232 @@
+envs::{
```

Comment:
The formatting of the text `.ion` files in this PR use IonJava's default text pretty printer. This is different than the previous formatting used in the ported spec tests PR #30.

Currently working on an Ion linter for these tests as part of #31, which should address any of the `.ion` text style concerns.